modules/Markbook/src/MarkbookView.php

```diff
@@ -130,6 +130,7 @@ public function __construct(Core $gibbon, Connection $pdo, $gibbonCourseClassID,
         $this->pdo = $pdo;
 
         $this->gibbonCourseClassID = $gibbonCourseClassID;
+        global $session;
```

Comment:
Same as above.

For readability sake, the defacto standard is to place the "global" statement at the top of a function.